val/src/avs_exerciser.c

```diff
@@ -234,6 +234,10 @@ val_exerciser_execute_tests(uint32_t level)
       }
   }
 
+  if (g_single_module != SINGLE_MODULE_SENTINEL && g_single_module != AVS_EXERCISER_TEST_NUM_BASE) {
+    return AVS_STATUS_SKIP;
```

Comment:
Add log print for consistency in log
val_print(AVS_PRINT_TEST, "      USER Override - Skipping all Exerciser tests (running only a single module)\n", 0);

Reply:
Fixed. Thanks for pointing these out.